product_packaging_level/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import product_packaging, product_packaging_level
```

Comment:
Your code is fine with a fresh empty database, no problem.

But in the case of my project in production with many modules installed I've this error
![level](https://github.com/OCA/product-attribute/assets/1853434/58eb2ec8-2f3c-4004-aea1-cdf6a8d6841a)

```suggestion
from . product_packaging_level, import product_packaging
```

With this suggestion I've no more error because product_packaging_level table is already a created 
when trying to define packaging_level_id m2o.

This is a really rare case that I have encountered regarding the import order of odoo models for about ten years

Reply:
@bealdav Done

Reply:
Thanks a lot, good job